tests/ignite/test_utils.py

```diff
@@ -174,6 +174,30 @@ def test_override_setup_logger(capsys):
     logging.shutdown()
 
 
+@pytest.mark.parametrize("encoding", [None, "utf-8"])
+def test_setup_logger_encoding(encoding, dirname):
+
+    fp = dirname / "log"
+    logger = setup_logger(name="logger", filepath=fp, encoding=encoding, reset=True)
+    test_words = ["hello", "你好", "привет"]
+    for w in test_words:
+        logger.info(w)
+    logging.shutdown()
+
+    with open(fp, "r") as h:
```

Comment:
We maybe need to read with `encoding="utf-8"` here as well

Reply:
Actually, Windows CI is failing the test: https://github.com/pytorch/ignite/actions/runs/8810663674/job/24191728286?pr=3240#step:15:2167
```
=========================== short test summary info ===========================
\x1b[31mFAILED\x1b[0m tests/ignite/test_utils.py::\x1b[1mtest_setup_logger_encoding[None]\x1b[0m - AssertionError: assert 1 == 3\n +  where 1 = len(['2024-04-24 09:08:44,276 logger INFO: hello\\n'])\n +  and   3 = len(['hello', '\u4f60\u597d', '\u043f\u0440\u0438\u0432\u0435\u0442'])
= 1 failed, 1842 passed, 72 skipped, 965 deselected, 15 warnings in 495.12s (0:08:15) =
```

@sjiang95 can you check this please ?